messages/rules-command.md

```diff
@@ -1,27 +1,71 @@
 # command.summary
 
-*DRAFT*: Command summary
+List the rules that are available to analyze your code.
 
 # command.description
 
-*DRAFT*: Command description
+You can also view details about the rules, such as the engine it's associated with, tags, and description.
+
+Use this command to determine the exact set of rules to actually analyze your code. The "code-analyzer run" command has the same flags as this command, so once you've determined the flag values for this command that list all the rules you want to run, you specify the same values to the "run" command.
+
+Use the --rule-selector flag to narrow down the list of rules based on specific criteria. For example, you can filter by engine, such as the rules associated with the RetireJS or ESLint engine. Or filter by the severity of the rules, such as high or moderate. Every rule has a tag, such as Recommended, and a name, and you can further filter using both. 
```

Comment:
Question: Is it information overload to have all of this in the command description instead of the flag-level description?

Reply:
Another good thing to discuss.  I personally like having a fair amount of info in the command description, just because it's all together and I can wrap my head around it all at once.  But yep, that can also be info overload for sure.  So I can move it all to the flag descriptions -- that way all the flags will be consistent in having both a summary and a description.  And we can decide in our meeting what's the most important stuff to have in the command description.